Planner/src/Task.java

```diff
@@ -1,12 +1,20 @@
 import java.util.ArrayList;
+import java.util.Date;
+import java.text.DateFormat;
+import java.time.format.DateTimeFormatter;
+
 
 public class Task {
 	String nameTask;
 	String descriptionTask;
+	String startTask;
+	String endTask;
```

Comment:
The start and end should be of type DateTime and you can rename them in startDate or startTime, endDate or endTime.